src/defines.h

```diff
@@ -75,6 +75,19 @@
   #define IMU_HAS_MAG false
   #define I2C_SPEED 400000
   #define IMU_MPU6050_RUNTIME_CALIBRATION // Comment to revert to startup/traditional-calibration
+#elif IMU == IMU_ICM20948
```

Comment:
Incompatible with current main branch. There are no IMU-specific defines anymore, if you wnat some defines for debug purposes, please move them to `debug.h`